backend/src/main/java/com/hyooun/wilson/review/domain/ReviewResult.java

```diff
@@ -0,0 +1,14 @@
+package com.hyooun.wilson.review.domain;
```

Comment:
ReviewResult 클래스에 대한 설명이 필요함

```suggestion
ReviewResult 클래스 설명 추가
```